Example/android/app/src/main/java/com/swmansion/rnscreens/example/MainApplication.java

```diff
@@ -5,6 +5,7 @@
 import android.content.Context;
 import com.facebook.react.PackageList;
 import com.facebook.react.ReactApplication;
+import com.reactnativerestart.RestartPackage;
```

Comment:
Is this needed at all? All packages should be autolinked iirc.

Reply:
Resolved by https://github.com/software-mansion/react-native-screens/pull/827/commits/3d2870e2ffe56635be5e5bf148359d1262360450